documentation/docs/workflow.md

```diff
@@ -0,0 +1,149 @@
+
+#### Feature (feat/, doc/, chore/, fix/)
+Singular PR contributions should create either `feat`, `doc` or `chore` branches, depending on the type of work to be done. `feat`, `doc` and `chore` branches may be branched from either the `dev` branch or an `epic` branch. If the number of lines of code is going to be relatively small and the work will be completed in a single PR, the branch should be created from `dev` and merged back into `dev` once completed. Otherwise, the branches should be created from their associated `epic` branch and be merged back into the same `epic` branch. 
+
+Any `feat` branch should include updates to the documentation and examples related to the feature. 
+It might be necessary to create a dedicated `doc` PR to, for example, catch up on documenting a feature. These `doc` PRs should be kept relatively small in order to not burden a reviewer with too many documentation updates. For example, during the documentation catch-up, we will have a branch/PR per documentation page.
+`chore` branches are short lived branches that contain no significant features or code updates, but rather smaller fixes such as typo's, code fixes and CI changes.
+
+It is recommended to integrate `dev` or `epic` regularly, depending on from where the branch started, to reduce the possibility and potential size of merge conflicts.
+
+#### Epic (epic/)
+Long-lived `epic` branches should be created and pushed to `origin` as soon as a feature is expected to require more than one PR. The `epic` branch should be branched from `dev` and should only accept merges that are related to the feature being developed. A PR should be opened as soon as the branch is created to publicly notify contributors about the development, the goals and requirements of the feature and the existence of the branch. It is recommended to integrate `dev` often to reduce the possibility and potential size of merge conflicts.
+
+#### Release (release/) {#release}
+Release branches allow to move changes from `dev` to `main`. They must be created from `dev`. If version strings need to be updated in files (like `package.json` e.g.) this should happen in commits on this branch. These branches need to be merged to `main` and `dev`. Since release branches will lead to a [release](#Release) they must contain a [changelog](#Changelog) and must be versioned.
+
+#### Hotfix (hotfix/) {#hotfix}
+Hotfix branches allow to fix critical bugs on the `main` branch, without the need to merge them in the `dev` branch first. They must be created from `main`. If version strings need to be updated in files (like `package.json` e.g.) this should happen in commits on this branch. These branches need to be merged to `main` and `dev`, so that both branches contain the fix. Since hotfix branches will lead to a [release](#Release) they must contain a [changelog](#Changelog) and must be versioned.
```

Comment:
I think hotfix branches are going to be replaced by support branches? If not, we still need to mention support branches somewhere.